src/core/manifest.js

```diff
@@ -133,14 +156,19 @@ async function buildManifest (env, opts = {}) {
     children: navtreeExternal,
   } = await _walk(opts.reposDir, ignoredUnderscores)
 
+  // Combine and sort local and external nav trees
+  const navtree = [...navtreeLocal, ...navtreeExternal]
+    .sort((a, b) => {
```

Comment:
Why does it get sorted twice? Once here and once on line 138.

Reply:
The first is only sorting children, not the top level folders—I abstracted it out into a single sorting function.